proposals/dao/tip_121c.ts

```diff
@@ -78,6 +85,54 @@ const validate: ValidateUpgradeFunc = async (addresses, oldContracts, contracts,
     '(millions)'
   );
   console.log('simpleFeiDaiPSM FEI balance', (await contracts.fei.balanceOf(addresses.simpleFeiDaiPSM)) / 1e18);
+
+  // 0. Verify PCV equity diff is 0
+  expect(eqDiff).to.equal(0);
+
+  // 1. Verify old DAI PSM is deprecated
+  // Has no funds
+  expect(await contracts.fei.balanceOf(addresses.daiFixedPricePSM)).to.equal(0);
+  expect(await contracts.dai.balanceOf(addresses.daiFixedPricePSM)).to.equal(0);
+
+  // Fully paused
+  expect(await contracts.daiFixedPricePSM.paused()).to.be.true;
+  expect(await contracts.daiFixedPricePSM.redeemPaused()).to.be.true;
+  expect(await contracts.daiFixedPricePSM.mintPaused()).to.be.true;
+
+  // 2. Verify new DAI PSM has DAI and no FEI (should have been burned)
+  expect(await contracts.dai.balanceOf(addresses.simpleFeiDaiPSM)).to.be.bignumber.greaterThan(MIN_DAI_ON_NEW_PSM);
```

Comment:
imo this can be more precise.  fei.totalSupply() - fei.balanceOf()/pcvDeposit.balance() of the timelock and pool 79 FEI

Reply:
Done. Atm have a check that DAI on the PSM is > user circulating FEI supply I measure - can't check equality as it's changing slightly what with the active old DAI PSM